flexget/plugins/input/html.py

```diff
@@ -44,6 +44,7 @@ def validator(self):
         advanced.accept('text', key='password')
         advanced.accept('text', key='dump')
         advanced.accept('text', key='title_from')
+        advanced.accept('boolean', key='allow_empty_links')
```

Comment:
I'm not sure the name properly conveys the meaning

Reply:
I'm open to any suggestion for a better name...